src/assets/scss/_settings.scss

```diff
@@ -498,6 +498,17 @@ $button-smaller-medium-shadow: 0 2px $button-background-hover;
 	opacity: 0.3;
 }
 
+// Hide visually, but not from screenreaders
```

Comment:
Foundation already has a utility class for this. Not sure if we want to use it or write our own